cmd/mongo-auth/replset_key.txt

```diff
@@ -0,0 +1 @@
+"replica set key"
```

Comment:
Is this just dummy data for testing/running locally?

Reply:
yes, it seems that with the new db version mongod requires a secret key for members of the replica set to communicate with each other